ambry-router/src/main/java/com/github/ambry/router/PutOperation.java

```diff
@@ -580,14 +580,10 @@ private void onChunkOperationComplete(PutChunk chunk) {
       logger.trace("{}: Successfully put data chunk with blob id : {}", loggingContext, chunk.getChunkBlobId());
```

Comment:
I believe you don't need this log any more.

Reply:
fixed.